contracts/AnchoredView/AnchoredView.sol

```diff
@@ -27,17 +27,17 @@ interface AnchorOracle {
  * @author Compound Labs, Inc.
  */
 contract AnchoredView is SymbolConfiguration {
-    /// @notice The mapping of posted by source prices per symbol
+    /// @notice The mapping of posted by reporter prices per symbol
     mapping(string => uint) public _prices;
 
     /// @notice circuit breaker for using anchor price oracle directly
     bool public breaker;
```

Comment:
Bit confusing to me that these are both breakers but one is called `breaker`

Reply:
renamed